### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mohdumar786**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ agree with how they affect the website.
 Installation
 ------------
 
-__1. Clone the Repo__
+__1. Clone the Repository
```
Comment (keeferrourke) [inline README.md:28]: This is syntactically incorrect markdown.

**Review by keeferrourke — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (keeferrourke) [reply README.md:28]: This also feels kind of spammy considering it is officially Hacktober, and this is not a meaningful change :(

**Comment by keeferrourke** [comment]
To other maintainers, I'd suggest **not** merging this change until at least November, even if the author does fix their syntax error.

**Comment by Fogest** [comment]
I see the user has also modified some other repo's README files with low-quality edits as well.

@mohdumar786 I should stress that low-quality pull requests do not qualify for hacktober per their rules:

> In line with Hacktoberfest value #2 (Quantity is fun, quality is key), we have provided examples of pull requests that we consider to be low-quality contributions (which we discourage).
> * Something that's clearly an attempt to simply +1 your pull request count for October.

https://hacktoberfest.digitalocean.com/details#rules

To the maintainers, I suggest marking this as a spammy pull request so that Github is advised of the abuse:

> Spammy pull requests can be given a label that contains the word "invalid" or "spam" to discount them. Maintainers are faced with the majority of spam that occurs during Hacktoberfest, and we dislike spam just as much as you. If you're a maintainer, please label any spammy pull requests submitted to the repositories you maintain as "invalid" or "spam", and close them. Pull requests with this label won't count toward Hacktoberfest.

**Comment by keeferrourke** [comment]
@Fogest thanks for your comment, I'm going to close this PR under the suspicion that this is hacktoberfest spam.

**Pull request closed without merging**